samples/place-class/style.scss

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ @use 'sass:meta'; // To enable @use via meta.load-css and keep comments in order
+
+ /* [START maps_map_simple] */
```

Comment:
I think this will conflict with the existing region tag for `maps_map_simple` in https://github.com/googlemaps/js-samples/blob/main/samples/map-simple/index.ts#L7-L23

Was there a reason you wanted to add a region tag to the scss file?

Reply:
ohhh good catch!